torchbox/torchbox.com
# Breadcrumb styling and tweak to pattern library structure

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1346362796)

### Description of Changes Made

- Adds styling for the breadcrumb, including making sure the forward slash is not read out by a screen-reader
- Moves the navigation components to a top-level folder to make them easier to locate in the pattern library
- Move the styleguide to be the first item in the pattern library so it is the first thing that loads
- Remove an unused sidebar tag that references an non-existent file
- Updates the styleguide
- Adds some reminders about the pattern library to the MR template.

### How to Test

With a local build running, visit either http://localhost:8000/pattern-library/render-pattern/patterns/navigation/components/breadcrumbs.html or any page in the site.

Ensure that at mobile only the immediate parent page shows.

### Screenshots

<details>
  <summary>Expand to see more</summary>

![Screenshot 2024-01-12 at 08 38 46](https://github.com/torchbox/torchbox.com/assets/771869/1151856b-9a37-4166-b2db-7eb53b34926f)
![Screenshot 2024-01-12 at 08 38 36](https://github.com/torchbox/torchbox.com/assets/771869/7a88ae10-6411-4258-97d5-e63ea9c8d9f7)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability
